lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -937,6 +937,36 @@ module.exports = class MethodMappings {
           const pageCdpConnection = await this.driver.createCDPConnection('page');
           await this.driver.register(username, password, pageCdpConnection);
 
+          return {
+            value: null
+          };
+        },
+
+        ///////////////////////////////////////////////////////////////////////////
+        // CDP commands
+        ///////////////////////////////////////////////////////////////////////////
+
+        async setGeolocation(coordinates) {
+          const pageCdpConnection = await this.driver.createCDPConnection('page');
+
+          await pageCdpConnection.execute(
+            'Emulation.setGeolocationOverride',
+            coordinates
+          );
+
+          return {
+            value: null
+          };
+        },
+
+        async clearGeolocation() {
+          const pageCdpConnection = await this.driver.createCDPConnection('page');
+
+          await pageCdpConnection.execute(
+            'Emulation.clearGeolocationOverride',
+            {}
+          );
```

Comment:
The above can also be done without creating a CDP connection (WS connection) by using `driver.sendDevToolsCommand`. Should we prefer CDP connections only or the latter command would also do (which is based on HTTP)?

Reply:
I spoke to @garg3133 about this and we are going to go with the `ws` connection instead.